mathics/doc/documentation/1-Manual.mdoc

```diff
@@ -49,28 +50,32 @@ Some of the features of \Mathics are:
+<section title="History">
+The version alpha version of \Mathics was done in 2011 by Jan Pöschko. He worked on it for a couple of years to about v0.5 which had 386 built-in symbols.
 
-Apart from performance issues, \Mathics has about about half of the features and libraries of \Mathematica.
+After that Angus Griffith took over primary leadership and rewrote the parser to pretty much the stage it is in now. He and later Ben Jones worked on it from 2013 to about 2017 to the v1.0 release. Towards the end of this period Bernhard Liebl worked on this mostly focused on graphics.
 
-Graphics has always been lagging and in the future we intend to decouple Graphics better so that the rich set of graphics packages that are out there can be more easily used.
+A <url>:docker image of the v.9 release: https://hub.docker.com/r/arkadi/mathics</url> can be found on dockerhub.
 
+The project was largely abandoned in its Python 2.7 state around 2017. Subsequently it was picked up by the current developers. A list of authors and contributors can be found in the
+<url>:<con>AUTHORS.txt</con>: https://github.com/Mathics-3/mathics/blob/master/AUTHORS.txt</url> file.
 </section>
 
-<section title="Who is behind it?">
-\Mathics was created by Jan Pöschko in 2011. From 2013 to about 2017 it had been maintained mostly by Angus Griffith and Ben Jones. Since then, a number of others have been people involved in \Mathics; the list can be found in the <con>AUTHORS.txt</con> file, <url>https://github.com/Mathics-3/mathics/blob/master/AUTHORS.txt</url>.
+<section title="What is missing?">
+There are lots of ways in which \Mathics could still be improved. <url>:<con>FUTURE.rst</con>: https://github.com/Mathics-3/mathics/blob/master/FUTURE.txt</url> has the current roadmap.
 
-If you have any ideas on how to improve \Mathics or even want to help out yourself, please contact us!
+While we always could use Python programming help, there are numerous other ways where we could use assistance.
 
-\skip
-Welcome to \Mathics, have fun!
-</section>
+<ul>
+<li>Ensure this document is complete and accurate. We could use help to ensure all of the Builtin functions described properly and fully, and that they have link to corresponding Wiki, Sympy, WMA and/or mpath links.
+Are the builtin summaries and examples clear and useful</li>
+<li>We could use help in LaTeX styling, and going over this document to remove overful boxes and things of that nature. We could also use help and our use of Asymptote. The are some graphics primitives such as for polyhedra that haven't been implemented. Similar graphics options are sometimes missing in Aymptote that we have available in other graphics backends.</li>
 
-</chapter>
+<li>write a grpahics backend favorite graphics backend: it could be a javascript library like jsiddle</li>
```

Comment:
Typo

Reply:
Thanks both should be corrected in 09596a10